db/sqlinterfaces.c

```diff
@@ -6563,6 +6563,7 @@ static void gather_connection_int(struct connection_info *c, struct sqlclntstate
         c->sql = NULL;
         c->fingerprint = NULL;
     }
+    c.in_transaction = clnt->in_client_trans;
```

Comment:
I think it should be `->` here.

Reply:
Semi-manual merge.  Thanks Rivers, that's an embarrassing one.